pallets/automation-time/src/benchmarking.rs

```diff
@@ -272,11 +295,14 @@ benchmarks! {
 
 	run_xcmp_task {
 		let caller: T::AccountId = account("caller", 0, SEED);
+		let currency_id: T::CurrencyId = T::GetNativeCurrencyId::get();
 		let time: u64 = 10800;
-		let task_id: T::Hash = schedule_xcmp_tasks::<T>(caller.clone(), vec![time], 1);
 		let para_id: u32 = 2001;
 		let call = vec![4,5,6];
-	}: { AutomationTime::<T>::run_xcmp_task(para_id.try_into().unwrap(), call, 100_000, task_id) }
+
+		T::XcmpTransactor::setup_chain_currency_data(para_id.clone(), currency_id.clone())?;
+		let task_id: T::Hash = schedule_xcmp_tasks::<T>(caller.clone(), vec![time], 1);
+	}: { AutomationTime::<T>::run_xcmp_task(para_id.clone().into(), caller, currency_id, call, 100_000, task_id.clone()) }
```

Comment:
can you add a verify here?

Reply:
I can try adding a verify for the sovereign account balance but the `xcm_send` function does not complete without error.

Reply:
discussed.  adding 2 writes and 1 read until we have benchmarking working with `send_xcm`.